scripts/release/build-experimental-typescript.js

```diff
@@ -1,22 +0,0 @@
-#!/usr/bin/env node
-
-/* eslint-disable import/no-commonjs */
-const fs = require('fs');
-const path = require('path');
-const shell = require('shelljs');
-
-const packageJsonPath = path.resolve('package.json');
-const packageJson = JSON.parse(fs.readFileSync(packageJsonPath).toString());
-const { version: currentVersion } = packageJson;
-const newVersion = `${currentVersion}-experimental-typescript.0`;
-packageJson.version = newVersion;
-packageJson.types = 'es/index.d.ts';
-fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson, null, 2));
-
-fs.writeFileSync(
-  path.resolve('src', 'lib', 'version.ts'),
-  `export default '${newVersion}';\n`
-);
-
-shell.exec(`NODE_ENV=production VERSION=${newVersion} yarn build`);
-shell.exec('yarn build:types');
```

Comment:
this shouldn't be removed

Reply:
This pull request is going to target a chore/ts-migration branch. It won't be merged on the stable unless the full migration is completed.